include/model.php

```diff
@@ -827,7 +775,13 @@ protected function get_language_terms_by_type() {
 	 */
 	protected function get_language_terms() {
 		add_filter( 'get_terms_orderby', array( $this, 'filter_language_terms_orderby' ), 10, 3 );
-		$post_languages = get_terms( array( 'taxonomy' => 'language', 'hide_empty' => false, 'orderby' => 'term_group' ) );
+		$post_languages = get_terms(
+			array(
+				'taxonomy'   => $this->translatable_objects->get_taxonomy_names( array( 'language' ) ),
+				'orderby' => 'term_group',
+				'hide_empty' => false,
```

Comment:
Alignment:
```suggestion
				'taxonomy'   => $this->translatable_objects->get_taxonomy_names( array( 'language' ) ),
				'orderby'    => 'term_group',
				'hide_empty' => false,
```

Reply:
Done in 6f3885e36e5950089a072d17901cf88f15275d96